src/community/json-ld/src/main/java/org/geoserver/jsonld/builders/SourceBuilder.java

```diff
@@ -0,0 +1,58 @@
+package org.geoserver.jsonld.builders;
```

Comment:
header required, yeah I know that is too much detail for a community module but I have the review tab open.

Reply:
fixed